source/community/reactnative/src/components/inputs/toggle-input.js

```diff
@@ -67,9 +68,14 @@ export class ToggleInput extends React.Component {
 						}
 						return (
 							<View
-								accessible={true}
-								accessibilityLabel={this.altText}
-								style={styles.toggleView}>
+								accessible={Platform.OS===Constants.PlatformIOS ? true : undefined}
+								accessibilityLabel={Platform.OS===Constants.PlatformIOS ? this.altText : undefined}
+								style={styles.toggleView}
```

Comment:
@frags51 , I have just one query here. Why we are setting `undefined` for Android?

Reply:
If we pass undefined, it will take the default value for a View, which is false.

Reply:
@vivekvijayakrishnan Updated the PR!